backends-common/elasticsearch-v8/src/main/java/org/apache/james/backends/es/v8/search/ScrolledSearch.java

```diff
@@ -93,9 +93,14 @@ private Mono<ScrollResponse<ObjectNode>> buildRequest(Optional<String> scrollId)
                     .scrollId(scrollId.get())
                     .scroll(TIMEOUT)
                     .build()))
-            //.orElseGet(() -> client.search(searchRequest));
-            .get(); // TODO: scroll and search return different responses now, thus this trick not working...
-                    // not sure of the best way to solve that yet
+            .orElseGet(() -> client.search(searchRequest)
```

Comment:
So actually here I kept the scroll for now and just adapted the code to quickly migrate...

However I believe though we can optimize this greatly in an other task/PR. 

https://www.elastic.co/guide/en/elasticsearch/reference/8.2/paginate-search-results.html#scroll-search-results => stating scrolling is not recommended anymore for deep pagination.

Instead they advice to use the search_after param in the search method: https://www.elastic.co/guide/en/elasticsearch/reference/8.2/paginate-search-results.html#scroll-search-results

That would require a little refactoring.

Reply:
Maybe we could create a ticket "evaluate PIT + scroll_after" ?

Reply:
> Regarding search_after, requesting a point in time looks expensive and likely is a blocker. Before adoption I'd like to have a performance evaluation...

I'm not sure it's more expensive than requesting a scrollId... Felt a bit the same mechanism to me, you have it for the time of your scroll and then you clear it. 

Agreed that it should be subject to a performance tests of course before adoption :)